docs/contribute/Release-Scenarioo-Viewer-Web-Application.md

```diff
@@ -79,15 +85,30 @@ Usually this should be the same as the Scenarioo version that you are going to r
 
 ### Publish the Release on Github
 
+* Fast forward master to release
+    ```
+    git checkout master
+    git merge --ff-only release/<version>
+    git push
+    ```
+  
+* Merge release branch into develop (if not done yet)
+    ```
+    git checkout develop
+    git merge release/<version>
+    git push
+    ```
+    
+* **DO NOT delete the release branch. We will keep the release branch to do improvements on the documentation for the just released version (or if ever needed: for hotfixes as well)**. Because usually short after a release we get feedback about how to improve that version of the docu. Such things have still to be fixed on the release branch and merged both to master and develop, such that those changes are in the published documentation of that version!
 * Create a release on github to make the release publicly visible: https://github.com/scenarioo/scenarioo/releases. 
 * For description use the release notes of that release version from CHANGELOG.md in the release description on github.
 * Make sure you safe it first only as a draft and do not publish them yet.  
 * Attach the binary of the release (WAR-file) to the release notes (see area "Attach binaries by dropping them here ...").
 * Publish the release when done.
 
-### Publish the Release on Maven Central
+### Publish the Release as signed WAR on Maven Central
```

Comment:
Do we really do this?

Reply:
we didnt so far - something we need to discuss - you are right, but I didnt add it 
@danielsuter